banzai/images.py

```diff
@@ -19,7 +19,10 @@
 class Image(object):
 
     def __init__(self, pipeline_context, filename=None, data=None, header={},
-                 extension_headers=[], bpm=None):
+                 extension_headers=None, bpm=None):
```

Comment:
Can the signature just set the default value for `extension_headers` like `extension_headers=[]`?

Reply:
No actually. This is a nasty Python feature. 
See http://docs.python-guide.org/en/latest/writing/gotchas/